Source/SwiftLintFramework/Rules/Lint/PrivateUnitTestRule.swift

```diff
@@ -156,6 +192,77 @@ private class Visitor: SyntaxVisitor, ViolationsSyntaxVisitor {
     }
 }
 
+private class Rewriter: SyntaxRewriter, ViolationsSyntaxRewriter {
+    private(set) var correctionPositions: [AbsolutePosition] = []
+    let locationConverter: SourceLocationConverter
+    let disabledRegions: [SourceRange]
+    private let parentClassRegex: NSRegularExpression
+
+    init(parentClassRegex: NSRegularExpression,
+         locationConverter: SourceLocationConverter,
+         disabledRegions: [SourceRange]) {
+        self.locationConverter = locationConverter
+        self.disabledRegions = disabledRegions
+        self.parentClassRegex = parentClassRegex
+    }
```

Comment:
Nit: keep the same order as the parameters?

```suggestion
        self.parentClassRegex = parentClassRegex
        self.locationConverter = locationConverter
        self.disabledRegions = disabledRegions
    }
```